content/blog/data-processing-compliance-statements-and-sla.md

```diff
@@ -49,6 +49,20 @@ In particular, any software that is released will adhere to the [release policy]
 
 You may either complete various certifications in house, by using a third party or by procuring the software from a commercial entity that provides such services. We cannot make any particular recommendations, but if that is relevant to you that might be something to seek out.
 
+# Provenance and export restrictions
+
+As a Delaware incorporated 501(c)3 organisation the Apache Software Foundation
+labours under the rules and regulation of the United States. We have a
+[separate page](https://www.apache.org/licenses/exports/) dedicated to detailed
+information on the export control status of the Apache Software Foundation's
+products.
```

Comment:
Perhaps make this "releases".

Or releases you receive under the Apache software license agreement.

To curtail this very much to the official releases & under that agreement/license, etc.

Reply:
> Perhaps make this "releases".

applied :+1: 

> Or releases you receive under the Apache software license agreement.

(I've kept it to just 'releases' as we don't make any promises about software *other* people distribute under the Apache license, and clarifying that would make the sentence rather elaborate ;) )